### DISCUSSION TITLE
How to get all mime types for mime group?

### DISCUSSION THREAD

**Discussion by Boscop** [conduit-rust/mime-types#21] [Q&A]
How to query this crate to list all mime types that belong to a given group (specified by string, e.g. the `"audio/*"` or `"image/*"` group)?
(This would be very useful for web scrapers / bulk downloaders.)
(Also it would be useful to be able to get the group of a given mime type.)

**Comment by Turbo87** [comment]
There is currently no API available in this crate that allows this. PRs for that are welcome though.
